modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -306,6 +309,31 @@ And owners with:
 
 ----
 chown postgres:postgres /var/lib/containers/storage/volumes/var-pgsql
-chown wwwrun:www /var/lib/containers/storage/volumes/var-spacewalk
+chown wwwrun:www        /var/lib/containers/storage/volumes/var-spacewalk
 ----
 
+
+
+== Logical Volume Management (LVM)
+
+
+// container-deployment/suma/server-deployment-vm-suma.adoc etc.
+For all kind of virtual machines (VM), LVM is generally not needed and not recommended.
+The disk setup is virtual and separate disks for subvolumes are possible and recommended.
+
+// container-deployment/suma/server-deployment-suma.adoc
+For other deployments, separate disks for subvolumes are also recommended.
+
+On the {productname} Server, the [command]``mgr-storage-server`` tool does it for the database and spacewalk volumes [literal]``var-pgsql`` and [literal]``var-spacewalk``.
+On the {productname} Proxy, the [command]``mgr-storage-proxy`` tool does it for the cache volume [literal]``var-cache``.
```

Comment:
This is not correct.
First, we have much more podman volumes then `var-spacewalk`. That is why `mgr-storage-server` moves complete `/var/lib/containers/storage/volumes` [see source](https://github.com/uyuni-project/uyuni/blob/master/uyuni/uyuni-storage-setup/scripts/mgr-storage-server#L48C1-L48C17).
When database is requested to be moved to it's disk, it then moves [one more subdir](https://github.com/uyuni-project/uyuni/blob/master/uyuni/uyuni-storage-setup/scripts/mgr-storage-server#L67)

Similarly on proxy, it's not only `var-cache`, [but complete volume dir](https://github.com/uyuni-project/uyuni/blob/master/uyuni/uyuni-storage-setup/scripts/mgr-storage-proxy#L45).

Also is there a reason insisting on naming it as `volumes` here?

I know that they are podman volumes, but it would be much more understandable if we just talk about absolute directories. Also LVM has volumes as well, so people can get quickly confused.

I am now looking through this file and it's not great. We absolutely have to instruct people to create more volumes than just `var-spacewalk`. For example retail customers have all their images in `srv-www`. The same autoinstallation distributions.

Reply:
ok, I use now absolute directory names.

Your last para is still an open question for me. I'll check the feedback in the channel (Cedric's comments).

Reply:
www is now also part of the server table.